lib/huginn_acumen_product_agent/concerns/inv_product_query_concern.rb

```diff
@@ -44,10 +44,11 @@ def process_inv_product_response(raw_data, digital_format_list)
 
         product['@type'] = 'Product'
         product['isTaxable'] = get_field_value(p, 'Inv_Product.Taxable') == '1'
-        product['productAvailability'] = get_field_value(p, 'Inv_Product.Not_On_Website') == '0'
```

Comment:
We probably need to remove all of the other places productAvailability gets set if we are going to be handling this in one place.